src/main/java/org/tom_v_squad/soiwenttoaconcert/controllers/ArtistController.java

```diff
@@ -0,0 +1,60 @@
+package org.tom_v_squad.soiwenttoaconcert.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.*;
+import org.tom_v_squad.soiwenttoaconcert.data.ArtistRepository;
+import org.tom_v_squad.soiwenttoaconcert.models.Artist;
+import org.tom_v_squad.soiwenttoaconcert.models.Venue;
+
+import javax.validation.Valid;
+import java.util.Optional;
+
+@Controller
+@RequestMapping("artists")
+public class ArtistController {
+
+    @Autowired
+    private ArtistRepository artistRepository;
+
+    @RequestMapping("")
+    public String index(Model model) {
+        model.addAttribute("title", "Artists");
+        model.addAttribute("artists", artistRepository.findAll());
+
+        return "Artist/index";
```

Comment:
Is `Artist` a directory name in this template?  Best practice would be to keep it lowercase.  Rename that to `artist` (the directory) and refactor this controller with the new name.